contracts/governance/TimelockController.sol

```diff
@@ -169,28 +169,6 @@ contract TimelockController is AccessControl, ERC721Holder, ERC1155Holder {
         return getTimestamp(id) > 0;
     }
 
-    /**
-     * @dev Returns whether an operation is pending or not. Note that a "pending" operation may also be "ready".
-     */
-    function isOperationPending(bytes32 id) public view virtual returns (bool) {
-        return getTimestamp(id) > _DONE_TIMESTAMP;
-    }
-
-    /**
-     * @dev Returns whether an operation is ready for execution. Note that a "ready" operation is also "pending".
-     */
-    function isOperationReady(bytes32 id) public view virtual returns (bool) {
-        uint256 timestamp = getTimestamp(id);
-        return timestamp > _DONE_TIMESTAMP && timestamp <= block.timestamp;
-    }
-
-    /**
-     * @dev Returns whether an operation is done or not.
-     */
-    function isOperationDone(bytes32 id) public view virtual returns (bool) {
-        return getTimestamp(id) == _DONE_TIMESTAMP;
-    }
-
```

Comment:
Maybe we should keep these function for backward compatibility. Some interfaces may rely on them.

@ernestognw do you know if defender uses that ?

Reply:
Indeed they use them, though I'm in favor of removing them. I think Defender can just wait to implement 5.0 feature detection. This is something we'll need to discuss next Monday on sync.